tests/src/test/scala/whisk/core/database/test/ReplicatorTests.scala

```diff
@@ -52,6 +52,7 @@ class ReplicatorTests
 
   val replicatorClient =
     new ExtendedCouchDbRestClient(dbProtocol, dbHost, dbPort.toInt, dbUsername, dbPassword, "_replicator")
+  replicatorClient.createDb()
```

Comment:
why is this new?

Reply:
In this version of CouchDB, the `_replication`-DB does not exist. So the initial cleanup fails with 404.